app/src/main/res/values/strings.xml

```diff
@@ -171,91 +171,92 @@
     <string name="github_sponsors">GitHub Sponsors</string>
-    <string name="donate_desc">Support EVMap\'s development with a one-time donation</string>
-    <string name="github_sponsors_desc">Support EVMap on GitHub Sponsors</string>
+    <string name="donate_desc">Make a one-time donation to support EVMap development</string>
+    <string name="github_sponsors_desc">Support EVMap</string>
     <string name="unnamed_filter_profile">Unnamed filter profile</string>
     <string name="privacy_link">https://evmap.vonforst.net/en/privacy.html</string>
     <string name="faq_link">https://evmap.vonforst.net/en/faq.html</string>
     <string name="chargeprice_faq_link">https://evmap.vonforst.net/en/chargeprice_faq.html</string>
     <string name="required">required</string>
     <string name="edit_filter_profile">Edit “%s”</string>
-    <string name="pref_search_delete_recent">Delete recent search results</string>
-    <string name="deleted_recent_search_results">Recent search results have been deleted</string>
-    <string name="settings_data_sources">Data sources</string>
+    <string name="pref_search_delete_recent">Delete recent searches</string>
+    <string name="deleted_recent_search_results">Recent searches deleted</string>
+    <string name="settings_data_sources">Sources</string>
     <string name="help">Help</string>
     <string name="settings_android_auto">Android Auto</string>
-    <string name="pref_chargeprice_allow_unbalanced_load">Enable unbalanced load</string>
-    <string name="pref_chargeprice_allow_unbalanced_load_summary"><![CDATA[Allow charging with >4.5 kW at AC stations for cars with single-phase charger]]></string>
-    <string name="pref_map_rotate_gestures_enabled">Enable map rotation</string>
-    <string name="pref_map_rotate_gestures_on">Map can be rotated with two-finger gesture</string>
-    <string name="pref_map_rotate_gestures_off">Map will be fixed to north-up</string>
+    <string name="pref_chargeprice_allow_unbalanced_load">Unbalanced load</string>
+    <string name="pref_chargeprice_allow_unbalanced_load_summary"><![CDATA[Allow single-phase charging at over 4.5 kW on three-phase chargers.]]></string>
+    <string name="pref_map_rotate_gestures_enabled">Map rotation</string>
+    <string name="pref_map_rotate_gestures_on">Use two fingers to rotate the map</string>
+    <string name="pref_map_rotate_gestures_off">Rotation off (north always upwards)</string>
     <string name="refresh_live_data">refresh real-time status</string>
-    <string name="autocomplete_connection_error">Suggestions could not be loaded</string>
+    <string name="autocomplete_connection_error">Could not load suggestions</string>
     <string name="pref_language_device_default">Device default</string>
     <string name="pref_language_en">English</string>
     <string name="pref_language_de">German</string>
     <string name="pref_language_fr">French</string>
+    <string name="pref_language_nb-rNO">Norwegian Bokmål</string>    
```

Comment:
?

Reply:
I think @Altonss [convinced me](https://github.com/johan12345/EVMap/issues/208#issuecomment-1224774117) that as the list of languages grows it's not really useful to translate every language name into every language. So I think the best way to handle this is to move the `pref_language_*` strings (except `_device_default) to `donottranslate.xml` and make them all localized, i.e.:
```
<string name="pref_language_en">English</string>
<string name="pref_language_de">Deutsch</string>
<string name="pref_language_fr">Français</string>
<string name="pref_language_nb-rNO">Norsk Bokmål</string>    
```